Zilliqa/contracts/xsgd_contract.scilla

```diff
@@ -405,8 +405,10 @@ transition mint(to: ByStr20, value : Uint128, initiator : ByStr20, to_bal : Uint
   allowed_amount <- minterAllowed[initiator];
   match allowed_amount with
   | Some (Some a) =>
+    greater_than_zero = uint128_gt value zero;
```

Comment:
Oh, this would have been a problem right, let's say we try to mint negative amounts? 😰

Reply:
This is type `UintX` right? There is no negative `value`.